components/lane_follower/src/pyproject.toml

```
@@ -0,0 +1,8 @@
+[project]
+version = '1.0'
+name = "individual-follower"
+authors = [
+  {name = "Vaibhav Hariani"},
+  {name = "Isaiah Rivera"},
+]
+description = "IGVC lane follower module for a single camera"
```

```suggestion
description = "lane follower for a video feed"
```
Something along those lines.